org.thentrythis.Samplebrain.yaml

```diff
@@ -0,0 +1,97 @@
+id: org.thentrythis.Samplebrain
+runtime: org.kde.Platform
+runtime-version: "5.15-22.08"
+sdk: org.kde.Sdk
+
+rename-desktop-file: samplebrain.desktop
+rename-icon: samplebrain
+
+command: samplebrain
+
+finish-args:
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--socket=pulseaudio"
+  - "--device=dri"
+  - "--share=network"
+
+cleanup:
+  - "/include"
+  - "/lib/cmake"
+  - "/lib/pkgconfig"
+  - "/share/doc"
+
+modules:
+  - name: libsndfile
+    buildsystem: cmake-ninja
+    config-opts:
+      - "-DCMAKE_BUILD_TYPE=Release"
+      - "-DBUILD_SHARED_LIBS=ON"
+      - "-DENABLE_EXTERNAL_LIBS=NO"
+      - "-DENABLE_CPACK=NO"
+      - "-DBUILD_PROGRAMS=OFF"
+      - "-DBUILD_EXAMPLES=OFF"
+    sources:
+      - type: archive
+        url: https://github.com/libsndfile/libsndfile/releases/download/1.1.0/libsndfile-1.1.0.tar.xz
+        sha256: 0f98e101c0f7c850a71225fb5feaf33b106227b3d331333ddc9bacee190bcf41
+
+  - name: portaudio
+    buildsystem: autotools
+    config-opts:
+      - "--with-alsa"
+      - "--without-jack"
```

Comment:
any reason to no use jack (it's pipewire in the sandbox)?

Reply:
It doesn't seem like portaudio uses it even when I enable it, but I'll do it just in case.

Is there a special permission to use JACK in the sandbox? (Like `xdg-run/pipewire-0`?)